semgrep-monitor

```diff
@@ -50,11 +75,40 @@ def generate_message(notify_reason: str) -> Dict[str, Any]:
     ]
 
 
-def main():
+def report_start_to_semgrep_app():
+    if not args.semgrep_app_token:
+        return
```

Comment:
Some sort of error logging would be useful here

Reply:
Are you sure? I expect most users would want to run semgrep-action without reporting to semgrep app. If we add a warning, they'd get that message on every run.

Reply:
Maybe hide it behind a `--verbose`?